src/components/selections/select.js

```diff
@@ -377,6 +377,8 @@ function prepSelect(evt, startX, startY, dragOptions, mode) {
         throttle.done(throttleID).then(function() {
             throttle.clear(throttleID);
 
+            // Only points selected by the new selection are presented in eventData here
+            // Should we provide all the selected points instead?
```

Comment:
Yes, I think so 😄 (this is basically #4095, right?)
As it stands when you edit one of the selections later, the points from all selections are included; we should do that when you create a new selection as well.

Reply:
No, this `plotly_selected` one returns points similar to the `plotly_selecting` but it fires after a new selection not during the process of selecting. The shift/alt missing part is actually addressed in https://github.com/plotly/plotly.js/pull/6243/commits/b9d2db8788fb087d31a4a94b29332bb49d8a1065.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/6243/commits/8776d49fcd10a0524304a95efe1a5626a6978d11.